site/pages/docs/ref/eqht-css/eqht-css-fr.hbs

```diff
@@ -30,7 +30,34 @@
 
 <section>
 	<h2>Exemples pratiques</h2>
-	<p><a href="../../../demos/eqht-css/eqht-css-fr.html">Exemples pratiques du plugiciel d'égalisation des hauteurs pure css</a></p>
+	<p><a href="../../../demos/eqht-css/eqht-css-en.html">Exemple en anglais</a></p>
+	<p><a href="../../../demos/eqht-css/eqht-css-fr.html">Exemple en français</a></p>
+</section>
+
+<section>
+	<h2>Évaluation et rapport</h2>
+	<ul>
+		<li>Évaluation d'accessibilité
+			<ul>
+				<li><a href="../../../demos/eqht-css/reports/1-assessment-wcag21-fr.html">Rapport français</a></li>
+				<li><a href="../../../demos/eqht-css/reports/1-assessment-wcag21-en.html" hreflang="en">Rapport anglais</a></li>
+			</ul>
+		</li>
+		<li>Raport de conformance à l'accessibilité
+			<ul>
+				<li><a href="../../../demos/eqht-css/reports/1-acr-wcag21-fr.html">Rapport français</a></li>
+				<li><a href="../../../demos/eqht-css/reports/1-acr-wcag21-en.html" hreflang="en">Raport anglais</a></li>
+			</ul>
+		</li>
+	</ul>
+</section>
+
+<section>
+	<h2>Évaluation et rapport</h2>
+	<ul>
+		<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/eqht-css/reports/1-assessment-wcag21-en.html">Rapport Anglais</a></li>
+		<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/eqht-css/reports/1-assessment-wcag21-fr.html">Rapport Français</a></li>
+	</ul>
```

Comment:
The content is already there

```suggestion
```

Reply:
This change request is duplicated by another change request. The required change is still required